.travis.yml

```diff
@@ -37,7 +37,7 @@ matrix:
     - language: python
       os: linux
       python:
-        - 3.6
+        - "3.8"
```

Comment:
Weird, the build configuration had not been changed: https://github.com/apache/incubator-marvin/runs/4835434559

Reply:
It was getting stuck on validation.